developer_setup.md

```diff
@@ -148,49 +149,24 @@ git clone git@github.com:ManageIQ/manageiq.git
 cd manageiq
 ```
 
-## Configure Project Manually
+## Configure Project
 
-All the steps below are wrapped inside the `bin/setup` script, but they are outlined here to explain how to completely bootstrap a developer environment. If you've run PostgreSQL in a container, be sure to export the `DATABASE_URL` variable to connect to the container over TCP instead of a UNIX file socket.
-
-```bash
-export DATABASE_URL='posgresql://localhost:5432' # optional, only necessary if PostgreSQL is running in a container
-```
-
-### Create neccessary config files
-
-In order to run the application locally, a few sample config files need to be copied into their appropriate locations.
-
-```bash
-[[ -f "certs/v2_key" ]] || cp "certs/v2_key.dev" "certs/v2_key"
-[[ -f "config/cable.yml" ]] || cp "config/cable.yml.sample" "config/cable.yml"
-[[ -f "config/database.yml" ]] || cp "config/database.pg.yml" "config/database.yml"
-mkdir -p log
-```
-
-### Install Bundler and Gems
+A Ruby script called `setup` is available to quickly set up the application. This script installs the required Gems, sets up necessary JavaScript libraries, creates and migrates the database, and finally seeds the database with development content.
 
 ---
 **NOTE**
 
-macOS requires platform specific Gems. Run `bundle config specific_platform true` before installing.
+macOS requires platform specific Gems. Run `bundle config specific_platform true` before running `setup`.
 
-CentOS 7 requires the `rh-postgresql10` SCL environment in order to compile the `pq` Gem's native extensions. Run `bundle install` inside the `scl` environment: `scl enable rh-postgresql10 'bundle install'`.
+CentOS 7 requires the `rh-postgresql10` SCL environment in order to compile the `pq` Gem's native extensions. Run `setup` inside the `scl` environment: `scl enable rh-postgresql10 'bin/setup'`.
 
----
+If you've run PostgreSQL in a container, be sure to export the `DATABASE_URL` variable to connect to the container over TCP instead of a UNIX file socket.
 
 ```bash
-gem install bundler --version 1.16 --conservative
-bundle config specific_platform true # optional for non-macOS platforms
-bundle install
-bundle exec rails update:ui
-bundle exec rails db:create
-bundle exec rails db:migrate
-bundle exec rails db:seed
-bundle exec rails test:vmdb:setup
-bundle exec rails log:clear tmp:clear
+export DATABASE_URL='posgresql://localhost:5432' # optional, only necessary if PostgreSQL is running in a container
```

Comment:
Shouldn't that be the the IP / hostname of the container rather than `localhost`?

Reply:
Not in the podman command example I used above. I expose container port 5432 to the host on port 5432. That simulates as close as possible a PostgreSQL database running locally on the host.